src/components/atoms/Dropdown/Dropdown.scss

```diff
@@ -73,10 +77,22 @@
       padding: 0;
       margin: 0;
       opacity: 0;
+      visibility: visible;
+      // color: hsl(0, 0%, 20%);
```

Comment:
let's remove this

Reply:
I was trying to just copy the defaults from what the component generated in their custom classes. Still wary of removing them w/o first checking how it works as it's "should be".